clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestStrategy.java

```diff
@@ -29,4 +36,16 @@
 public class KubernetesManifestStrategy {
   Boolean versioned;
   Integer maxVersionHistory;
+
+  public static void useSourceCapacity(KubernetesHandler deployer, KubernetesManifest manifest, KubernetesV2Credentials credentials) {
```

Comment:
I would leave `useSourceCapacity` as a `Boolean` just like `versioned` in this class, and then read & write the value in the `KubernetesManifestAnnotator.java` just like is done for other values.